qa/tasks/cephfs/test_misc.py

```diff
@@ -61,12 +61,13 @@ def test_fs_new(self):
 
         self.fs.put_metadata_object_raw("key", dummyfile)
 
-        timeout = 10
+        def get_pool_df(fs, name):
+            try:
+                return fs.get_pool_df(name)['objects'] > 0
+            except:
```

Comment:
If we ever get around to running a python linter on these files, this bare `except:` will trigger it -- prefer to catch a named exception type.

Also returning None is more common than returning False in situations like this, but then we're being a bit un-pythonic by returning instead of raising, so meh.